examples/minimal-logs/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "ami_id" {
+  type        = string
+  description = "AMI to use for Tamr EC2 instance"
+}
+
+variable "name-prefix" {
+  type        = string
+  description = "A string to prepend to names of resources created by this example"
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "A map of tags to add to all resources created by this example."
+  default = {
+    Author      = "Tamr"
+    Environment = "Example"
+  }
+}
+
+variable "ingress_protocol" {
+  type        = string
+  description = "Protocol for ingress rules. If not icmp, icmpv6, tcp, udp, or all use the protocol number."
+}
+
+variable "egress_protocol" {
+  type        = string
+  description = "Protocol for egress rules. If not icmp, icmpv6, tcp, udp, or all use the protocol number."
+}
```

Comment:
If we can avoid duplicating these variables and pick reasonable defaults, let's do that.

Reply:
Fixed.